addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -315,6 +327,9 @@ MockServer.include({
         if (args.method === 'message_post') {
             return Promise.resolve(this._mockMessagePost(args));
         }
+        if (route === '/mail/read_followers') {
+            return this._mockFollowersRead(args);
```

Comment:
I meant also make `_performRpc` itself async to make it very obvious that this returns a promise?